src/components/blogs/Blogs.jsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+export const Blogs = () => {
+  return (
+    <div>
+      <Link to="/singleBolgPage">Lorem ipsem till Gunsu add her code</Link>
```

Comment:
routes are always lowercase

Reply:
@basil-kawak

Reply:
@Samah-Mustafa I believe you can do them like this ```<Route path="/singleblogpage" component={SingleBolgPage} />```